src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -100,20 +100,21 @@ protected internal override void RemoveCallbackForPinValueChangedEvent(int pinNu
 
         protected internal override void SetPinMode(int pinNumber, PinMode mode)
         {
-            bool failed  = false;
+            int requestResult  = -1;
```

Comment:
why reverting this?

Reply:
See https://github.com/dotnet/iot/pull/219#discussion_r256269017

Reply:
yeah @ahsonkhan showed that this is better